### PR TITLE
Break composed div col so we don't overflow

### PR DESCRIPTION
**Pull request by karelhala**
### Jira issue
RHCLOUD-1146

### Preview
Screenshot from 2019-06-13 11-03-58

### CODE REVIEW

**Comment by karelhala** [comment]
Nice catch @ryelo! Updated to use `break-all`

File: packages/inventory-insights/src/insights.scss [hunk 1]
```diff
@@ -46,7 +46,7 @@
         @include rem('margin-bottom', 10px);
         color: #333;
         word-break: break-all;
-        word-wrap: break-word;
+        word-wrap: break-all;
```
Comment (ryelo) [inline packages/inventory-insights/src/insights.scss]: This should be
`         word-wrap: break-word; `

Comment (karelhala) [reply packages/inventory-insights/src/insights.scss]: Ah shoot, I edited wrong file.

**Comment by karelhala** [comment]
Released in version [@redhat-cloud-services/frontend-components-inventory/v/0.0.15](https://www.npmjs.com/package/@redhat-cloud-services/frontend-components-inventory/v/0.0.15)

**Pull request merged**